.env.example

```diff
@@ -0,0 +1,3 @@
+GETDATE=https://plankton-app-lht9q.ondigitalocean.app/schedule
+EVENTS=https://plankton-app-lht9q.ondigitalocean.app/schedule
+EVENTID=https://plankton-app-lht9q.ondigitalocean.app/schedule/event/
```

Comment:
Not sure why we would do this.  Just have the url and each page should know the endpoint to call.